Description: fix qimage_conversions.cpp build on big endian systems
Origin: upstream, https://qt.gitorious.org/qt/qtbase/commit/404f4281fda764ca
Last-Update: 2014-12-16

--- a/src/gui/image/qimage_conversions.cpp
+++ b/src/gui/image/qimage_conversions.cpp
@@ -2252,7 +2252,7 @@
         convert_ARGB_to_ARGB_PM,
 #else
         0,
-        0
+        0,
 #endif
         0, 0, 0, 0
     }, // Format_RGBA8888
@@ -2281,7 +2281,7 @@
 #else
         0,
         0,
-        0
+        0,
 #endif
         0, 0, 0, 0
     }, // Format_RGBA8888_Premultiplied
